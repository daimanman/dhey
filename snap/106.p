{"C":1,"Duration":7568712692,"EndTime":1547004071395014900,"Err":"","FileId":"1","InputFileName":"","Method":"GET","N":10,"P":null,"Payload":"","QH":{},"Remark":"1222 test pg ","ReqNums":10,"StartTime":1547004063826302200,"Status":1,"TaskId":106,"Type":"FIXED","Url":"https://www.cwbaoguowang.com/images/open.png","Z":""}