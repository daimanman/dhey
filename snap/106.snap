{"AvgTotal":7.562701699999999,"Fastest":0.7031194,"Slowest":0.9235053,"Average":0.7562701699999999,"Rps":1.3222787829910043,"AvgConn":0.0262274,"AvgDNS":0.00049599,"AvgReq":0.00019375,"AvgRes":0.64896993,"AvgDelay":0.08087909000000001,"ConnMax":0,"ConnMin":0.0049599,"DnsMax":0,"DnsMin":0.0049599,"ReqMin":0.0010017,"ReqMax":0,"ResMin":0.0010017,"ResMax":0.4677469,"DelayMax":0.0668163,"DelayMin":0.1924827,"Lats":[0.9235053,0.7480645,0.7349706,0.7031194,0.7340706,0.7499098,0.7499947,0.749994,0.719077,0.7499958],"ConnLats":[0.0049599,0,0,0,0,0,0,0,0,0],"DnsLats":[0.0049599,0,0,0,0,0,0,0,0,0],"ReqLats":[0.0010017,0,0.0009358,0,0,0,0,0,0,0],"ResLats":[0.4677469,0.668276,0.6672185,0.6362961,0.6662562,0.6821286,0.6831733,0.683173,0.6522545,0.6831762],"DelayLats":[0.1924827,0.0797885,0.0668163,0.0668233,0.0678144,0.0677812,0.0668214,0.066821,0.0668225,0.0668196],"Total":7562701700,"ErrorDist":{},"StatusCodeDist":{"200":10},"SizeTotal":1350720,"SizeReq":135072,"NumRes":10,"LatencyDistribution":[{"Percentage":10,"Latency":0.719077},{"Percentage":25,"Latency":0.7349706},{"Percentage":50,"Latency":0.7499098},{"Percentage":75,"Latency":0.7499958},{"Percentage":90,"Latency":0.9235053},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.7031194,"Count":1,"Frequentcy":0.1},{"Mark":0.7251579899999999,"Count":1,"Frequentcy":0.1},{"Mark":0.74719658,"Count":2,"Frequentcy":0.2},{"Mark":0.76923517,"Count":5,"Frequentcy":0.5},{"Mark":0.7912737599999999,"Count":0,"Frequentcy":0},{"Mark":0.8133123499999999,"Count":0,"Frequentcy":0},{"Mark":0.83535094,"Count":0,"Frequentcy":0},{"Mark":0.85738953,"Count":0,"Frequentcy":0},{"Mark":0.87942812,"Count":0,"Frequentcy":0},{"Mark":0.90146671,"Count":0,"Frequentcy":0},{"Mark":0.9235053,"Count":1,"Frequentcy":0.1}],"TaskId":106,"Duration":7.5627017}